shell/ci/testing/setup-devenv.sh

```diff
@@ -10,6 +10,8 @@ source "$DIR/../../lib/logging.sh"
 source "$DIR/../../lib/github.sh"
 # shellcheck source=../../lib/box.sh
 source "$DIR/../../lib/box.sh"
+# shellcheck source=../../lib/mise.sh
```

Comment:
do you plan to keep this for future reference?

Reply:
@iamrz1 yes, why do you ask?